packages/kinvey-datastore2/src/datastore.js

```diff
@@ -1,12 +1,35 @@
+import isString from 'lodash/isString';
+import { Operation, OperationType } from './operation';
+
+export const DataStoreType = {
+  Cache: 'Cache',
+  Network: 'Network',
+  Sync: 'Sync'
+};
+Object.freeze(DataStoreType);
+
 export class DataStore {
-  /**
-   * Save an entity.
-   */
+  constructor(collection) {
+    if (!isString(collection)) {
+      throw new Error('Collection must be a string.');
+    }
+
+    this.collection = collection;
+  }
+
   save(entity, options) {
     return this.create(entity, options);
   }
 
   create(entity, options) {
-    throw new Error('Method of DataStore not implemented');
+    const operation = this._buildOperationObject(OperationType.Create, null, entity);
```

Comment:
I was really looking forward to seeing progress on this, rather than the tests, so I know how to proceed.